Framework/API/inc/MantidAPI/FileFinderHelpers.h

```diff
@@ -0,0 +1,41 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+#pragma once
+
+#include "MantidAPI/DllConfig.h"
+
+#include <string>
+#include <vector>
+
+namespace Mantid::API {
+
+class MANTID_API_DLL FileFinderHelpers {
```

Comment:
Good refactoring with this. There was no reason this stuff needed to be on an instance.

As you say I'm not super-keen on the name. Everything is instrument-related, right? Some ideas:

- `InstrumentFiles` -> `InstrumentFiles::getInstrumentFilename`
- `InstrumentFileHelpers` -> `InstrumentFileHelpers::getInstrumentFilename`

?